source/Sashimi.AzureWebApp/Endpoints/AzureWebAppEndpointValidator.cs

```diff
@@ -0,0 +1,14 @@
+using FluentValidation;
+
+namespace Sashimi.AzureWebApp.Endpoints
+{
+    class AzureWebAppEndpointValidator : AbstractValidator<AzureWebAppEndpoint>
+    {
+        public AzureWebAppEndpointValidator()
+        {
+            RuleFor(p => p.AccountId).NotEmpty().OverridePropertyName("Account");
+            RuleFor(p => p.WebAppName).NotEmpty().OverridePropertyName("WebApp");
```

Comment:
This used to be "Web App", which actually resulted in "Web  App" because FluentValidation adds an extra space